docs/_static/js/expand_tabs.js

```diff
@@ -21,17 +21,19 @@ $( document ).ready(function() {
 
   // Uses a hash referencing a Sphinx label from the URL page#sphinx-label
   var hash = window.location.hash.substr(1);
-  hash = hash.replace(/[^0-9a-z\-_]/gi, '');
-  // If the hash is inside a tab panel
-  var tab_with_reference = $(".sphinx-tabs-panel #" + hash).parents(".sphinx-tabs-panel");
+    if (hash) {
```

Comment:
What changed in this file? Seems like it needs a comment.

Reply:
The git diff is awful, it was just a matter of adding `if (hash) {` to fix a JS error occurring on all pages.

It's explained in line 22, the following code is pretty verbose:

```
// Uses a hash referencing a Sphinx label from the URL page#sphinx-label
var hash = window.location.hash.substr(1);
if (hash) {
...
```